packages/bpk-component-infinite-scroll/src/withInfiniteScroll.js

```diff
@@ -170,11 +170,11 @@ const withInfiniteScroll = <T: ExtendedProps>(
       // all visible data is fetched again (from 0 to current index) to update the list with the new data.
       // If after this call `isListFinished` is true, it means the new data source has no items and we need to
       // reset the list, which we do by setting `elementsToRender` to `[]` and `index` to `0`
-      const { isListFinished } = newState;
+      const { elementsToRender, index } = newState;
```

Comment:
Any specific reason for this change other than code style? I prefer the previous version as `isListFinished` is easier to read and understand.

Reply:
It wouldn't actually work the way it's currently written.
Fetch items now sets `isListFinished` to true when less than the requested elements are returned. The old code would have replaced those elements with an empty array because it saw `isListFinished` is true.

Reply:
Makes sense, can we update the inline comment to reflect this then, it is a bit confusing now.